arraySort.js

```diff
@@ -3,29 +3,37 @@ var oddNumbers = new Array();
 var characters = new Array();
 
 let numbers = {
-    evens: evenNumbers,
-    odds: oddNumbers,
-    chars: characters,
+    'evens': evenNumbers,
+    'odds': oddNumbers,
+    'chars': characters,
 }
 
 function arraySort(myArray) {
-    for (var i = 0; i < myArray.length; i++) {
-        if (isNaN(myArray[i])) {
-            characters.push(myArray[i]);
-        }
-        if (myArray[i] % 2 == 0) {
-            evenNumbers.push(myArray[i]);
-        }
-        if (myArray[i] % 2 == 1) {
-            oddNumbers.push(myArray[i]);
+
+    if (typeof myArray === 'string') {
+        return 'invalid input';
+    }
+    else if (typeof myArray === 'number') {
+        return 'invalid input';
+    }
```

Comment:
We it will be a bad practice to check for all invalid types e.g you didn't check for null and undefined among others. We can just make one check to confirm whether the input is not an array i.e
```js
if(!Array.isArray(myArray)){
 throw new Error('Invalid Input')
}
```